hazelcast/src/test/java/com/hazelcast/client/impl/spi/ClientProxyDestroyTest.java

```diff
@@ -82,4 +88,17 @@ public void testOperationAfterDestroy() throws Exception {
         clientMap.put(1, 1);
         assertEquals(1, clientMap.get(1));
     }
+
+    @Test
+    @Category(SlowTest.class)
```

Comment:
This test should not need to be slow. It does not use network.

Reply:
Ok changed.